ci/ansible/roles/pulp-crane/tasks/main.yml

```diff
@@ -22,17 +35,42 @@
     state: true
     persistent: true
 
-- name: Install Crane configuration file
-  copy:
-    src: crane.conf
-    dest: /etc/crane.conf
-    owner: root
-    group: root
-    mode: 0644
+- block:
+  - name: Install Crane configuration file
+    copy:
+      src: crane.conf
+      dest: /etc/crane.conf
+      owner: root
+      group: root
+      mode: 0644
+
+  - name: Open firewall port for Crane (firewalld)
+    firewalld:
+      port: "5000/tcp"
+      state: enabled
+      permanent: true
+      immediate: true
+  when:
+    - not (ansible_distribution == "RedHat" and ansible_distribution_major_version|int == 6)
```

Comment:
I'm advocating for conditions like:

```yaml
# RHEL 6
when:
  - ansible_distribution == "RedHat" and ansible_distribution_major_version == 6

# non-RHEL 6
when:
  - not (ansible_distribution == "RedHat" and ansible_distribution_major_version == 6)
```

One nice property of using this pair of conditions it covers every system we might target. For example, what if we target a CentOS 7 system? Or an OpenSUSE system? They'll both count as "non-RHEL 6", and the latter block of code will run.

On the other hand, this pair of conditions doesn't cover every system we might target:

```yaml
# RHEL 6
when:
  - ansible_distribution == "RedHat" and ansible_distribution_major_version == 6

# All of RHEL/Fedora/CentOS except RHEL 6, and not OpenSuse etc
when:
  - ansible_distribution == "RedHat" and ansible_distribution_major_version != 6
```